flask_rebar/swagger_generation/marshmallow_to_swagger.py

```diff
@@ -99,6 +89,30 @@ def _normalize_validate(validate):
         return validate
 
 
+def _get_property_name(name, field, context):
+    """
```

Comment:
(nit: rm leading \n from docstring (multiple occurrences))

Reply:
Haha I've always preferred the leading \n
I think this is one of those things where whichever one you're used to just _feels_ better.
Maybe we should consider adding something like https://github.com/ambv/black here, so we don't have to debate such things?